security.md

```diff
@@ -80,6 +81,15 @@ All security advisories made for Foreman are listed below with their correspondi
 
 ### Disclosure details
 
+#### <a id="2020-14334"></a>CVE-2020-14334: World readable cache could expose sensitive settigs
+Even encrypted settings have their raw values cached. Too permissive mode on cache dir caused, that anyone with access to the hosting system could read this encrypted settings.
+
+*Mitigation:* change cache directory mode `o-r`.
+
+* Affects RPM installations
```

Comment:
since which foreman version?

Reply:
It was part of https://github.com/theforeman/foreman-packaging/commit/847450fab873380fc50af885c67735f8f7abf05e which is titled `Import foreman`. Prior to that RPM specs were in foreman.git and was introduced in https://github.com/theforeman/foreman/commit/d30ac557d1d260bfdae13164cd83deca27c371cc. So my guess is Foreman 1.3 when using RPMs and the file cache backend. That means users of memcached are unaffected by this particular bug, though that does listen on a TCP port which is usually also accessible by local users.

Reply:
Thanks :heart: I was not able to find it out, I did not know about the switch to packaging repo :)

Reply:
Altough back then there might not be any sensitive information in the cache... :thinking:

Reply:
OAuth was added in https://github.com/theforeman/foreman/commit/fceb1f895c20019ad71863d23316a446c216845e, released in version 1.1 and already used settings. Caching of settings was added in https://github.com/theforeman/foreman/commit/d9a2ebac6e9ec0082e60265b794f54a29a1f7e65, also in 1.1.

Reply:
There are some passwords, those should be considered compromised as well if there was a possibility to use this vulnerability.
https://github.com/search?q=org%3Atheforeman+%22encrypted+%3D%3E+true%22&type=Code